contrib/scripts/aws-iam-create-yubikey-mfa.sh

```diff
@@ -1,28 +1,23 @@
 #!/bin/sh
 # Adds a Yubikey TOTP device to IAM
 
-set -eu
+set -e
```

Comment:
I'd prefer the safer `set -euo pipefail`

Reply:
`set -eo pipefail` makes sense, for the `-u` please read my other comment and tell me what you think